aspnetcore/security/authentication/configure-oidc-web-authentication.md

```diff
@@ -109,6 +109,22 @@ Add the OpenID Connect client settings to the application configuration properti
 },
 ```
 
+### Signed-out callback path configuration
+
+The <xref:Microsoft.AspNetCore.Builder.OpenIdConnectOptions.SignedOutCallbackPath%2A> (configuration key: "`SignedOutCallbackPath`") is the request path within the app's base path where the user agent is returned after sign out from the OIDC identity provider. Typically, the app's configuration relies on the default value of "`/signout-callback-oidc`".
+
+Configure the signed-out callback path in the app's OIDC provider registration. When using Microsoft Entra ID, set the path in the **Web** platform configuration's **Redirect URI** entries in the Entra or Azure portal:
+
+> :::no-loc text="https://localhost/signout-callback-oidc":::
+
+> [!NOTE]
+> A port isn't required for `localhost` addresses when using Microsoft Entra ID. Most other OIDC providers require the correct port.
+
+If you don't add the signed-out callback path URI to the app's registration in Entra, Entra refuses to redirect the user back to the app and merely asks them to close their browser window.
+
```

Comment:
I would remove the following note because this doc implements the default OIDC handler

Reply:
@damienbod ... When @halter73 provided the remark, it was in his OIDC sample (BFF pattern) for Blazor. I was about to also remove it from the BWA+OIDC article, but now I have second thoughts given that @halter73 provided it with an OIDC sample not using MS Identity Web. You/he will need to let me know ***exactly*** where the NOTE should appear. The three articles in question are ...

* Blazor+Entra
* Blazor+OIDC
* RP+OIDC (this is where I just removed it ... but with second thoughts given how @halter73 provided it in the first place in an OIDC sample)

Reply:
I agree with Damien that the main OIDC applications should omit most of the Entra-specific stuff.

At the top of the OIDC docs, we should have a note that links to the Entra variation of that doc for people using Entra, and those docs should include the link to https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/5783

The only reason I included it in the Blazor+OIDC sample app to begin with was because we didn't have an Entra-specific document for Blazor at that time.

Reply:
Thanks ... I'll get this resolved and merged tomorrow morning.

Reply:
Note that there is no Entra variation of the Razor Pages doc. We only have OIDC. If there's an Entra variation, it would be in Azure samples. I'll see if that's already cross-linked here. If not, I'll see if I can cross-link it on this PR.